autoware_map_msgs/msg/LaneletMapTileMetaData.msg

```diff
@@ -0,0 +1,6 @@
+# Metadata of lanelet map tile
+
+int32 tile_id
+string mgrs_code
+float64 origin_lat
+float64 origin_lon
```

Comment:
I think we should use projected x,y coordinates for this message so that client node that request the map doesn't have to subscribe to map_projector_info topic.

Reply:
I changes origin_lat and origin_lon fields with min_x, min_y, max_x and max_y

In previous version it just contain left bottom corner of lanelet tile but new version, I add also right top corner.